chef_master/source/server_security.rst

```diff
@@ -310,6 +310,108 @@ These newer add-ons will also write all of their secrets to ``/etc/opscode/priva
 
 While the file does not contain passwords in plaintext, it is not safe to share with untrusted users. The format of the secrets file allows Chef Infra Server deployments to conform to regulations that forbid the appearance of sensitive data in plain text in configuration files; however, it does not make the file meaningfully more secure.
 
+SSL Encryption Between Chef Infra Server and External PostgreSQL
+================================================================
+
+**New in Chef Server 13.1.13:**  Chef Server 13.1.13 introduces the capability to encrypt traffic between Chef Infra Server and an external PostgreSQL configuration over SSL.  These instructions are not all-encompassing and assume familiarity with PostgreSQL administration, configuration, and troubleshooting.  Consult the appropriate PostgreSQL documentation as necessary, e.g. https://www.postgresql.org/docs/9.6/ssl-tcp.html.
+
+To enable this encryption (typical scenario):
+
+1. Ensure that OpenSSL is installed on the PostgreSQL machine (consult appropriate documentation as necessary, e.g. https://www.openssl.org/).
+
+2. Ensure that PostgreSQL has SSL support compiled-in (this applies whether you are compiling your own or using a pre-compiled binary).
```

Comment:
Reworded this.

Reply:
Should this be 'compiled-in'  vs. 'compiled in' ?

Reply:
In this case "compiled in" is a phrasal verb so there's no hyphen. If it were "The compiled-in binary...." then it would be hyphenated because that's an adjective. To make matters more confusing, you can remove the hyphen if "compiled in" is an adjective that follows the noun. So "the binary is compiled in" is correct.